core/src/main/java/info/openrocket/core/preferences/ApplicationPreferences.java

```diff
@@ -21,10 +21,7 @@
 import info.openrocket.core.models.atmosphere.ExtendedISAModel;
 import info.openrocket.core.models.wind.PinkNoiseWindModel;
 import info.openrocket.core.preset.ComponentPreset;
-import info.openrocket.core.rocketcomponent.FlightConfiguration;
-import info.openrocket.core.rocketcomponent.MassObject;
-import info.openrocket.core.rocketcomponent.Rocket;
-import info.openrocket.core.rocketcomponent.RocketComponent;
+import info.openrocket.core.rocketcomponent.*;
```

Comment:
Please don't use wildcard imports. IntelliJ does this automatically. You can disable this: https://www.baeldung.com/intellij-disable-wildcard-import.